src/components/tfoot/tfoot-styles.jsx

```diff
@@ -6,10 +6,16 @@ export default theme => {
   return {
     tfoot: {
       ...mixins.basicBoxSizing,
-      display: 'block',
-      fontWeight: typography.fontWeightSemiBold,
       ...styleUtils.sizes(),
+      fontWeight: typography.fontWeightSemiBold,
       borderTop: `2px solid ${palette.shades.dark.text.muted}`,
+
+      '&$hiddenOnMobile': {
+        [mixins.maxMedia('md')]: {
+          display: 'none'
+        }
+      }
     },
+    hiddenOnMobile: {}
```

Comment:
You don't have to use the dollar signs here. It should be possible to move the contents from `&$hiddenOnMobile` to `hiddenOnMobile`.

Reply:
fixed